packages/atoms/src/Select.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { FlagIconName, Icon } from '@trezor/icons';
+
+import { Box } from './Box';
+import { Text } from './Text';
+import { BottomModal } from './Modal/BottomModal';
+import { SelectItem, SelectItemProps } from './SelectItem';
+
+export type SelectItem = {
+    label: string;
+    value: string;
+    iconName: FlagIconName;
+};
+
+type SelectProps = {
+    items: SelectItem[];
+    label: string;
+};
+
+const selectStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    backgroundColor: utils.colors.gray200,
+    borderWidth: utils.borders.widths.small,
+    borderRadius: utils.borders.radii.small,
+    borderColor: utils.colors.gray200,
+    color: utils.colors.gray600,
+    paddingLeft: 12,
+    paddingRight: 23.25,
+    height: 58,
+}));
+
+export const Select = ({ items, label }: SelectProps) => {
+    const { applyStyle } = useNativeStyles();
+    const [isOpen, setIsOpen] = useState<boolean>(false);
+    const [selectedItem, setSelectedItem] = useState<null | SelectItemProps['value']>(null);
```

Comment:
no internal state for values please, this will be handled by form lib

Reply:
Yeah I know, but I thought I'll just use internal state until we implement the form lib so it works for demo purposes and then we can refactor this? I'll be happy to do the use-form soon and then remove this.

Reply:
I want to avoid demo code anywhere as much as possible except Demo screen

Reply:
[fc5788e](https://github.com/trezor/trezor-suite/pull/5506/commits/fc5788e4a643dcc240d755543b1f56d03fefbea4)